Orange/widgets/unsupervised/owmds.py

```diff
@@ -187,135 +181,51 @@ def __init__(self):
         )
 
         form.addRow("Max iterations:",
-                    gui.spin(box, self, "max_iter", 10, 10 ** 4, step=1))
+                    gui.spin(box_optimization, self, "max_iter", 10, 10 ** 4, step=1))
 
         form.addRow("Initialization:",
-                    gui.radioButtons(box, self, "initialization",
+                    gui.radioButtons(box_optimization, self, "initialization",
```

Comment:
Why? IMHO, such temporary boxes can keep a short name `box`. We all understand it's a temporary into which a "paragraph" of a code puts the controls it's constructing. Making it more "descriptive" does not make the code cleaner, it just makes lines longer.